### PR TITLE
Move streaming to GA

### PR DESCRIPTION
**Pull request by jrodewig**
**IMPORTANT: DO NOT MERGE UNTIL STREAMING LAUNCH.**

Ticket(s): 
- [FE-4894](https://faunadb.atlassian.net/browse/FE-4894)
- [DOCS-2750](https://faunadb.atlassian.net/browse/DOCS-2750)

Merge all beta changes into main. Further documents streaming. Examples are borrowed from the [docs.fauna.com driver docs](https://docs.fauna.com/fauna/current/drivers/js-client#event-streaming-beta), but I re-tested them.

----
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

[DOCS-2750]: https://faunadb.atlassian.net/browse/DOCS-2750?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[FE-4894]: https://faunadb.atlassian.net/browse/FE-4894?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ptpaterson** [comment]
@jrodewig Adding the `pr/do-not-merge` label will cause checks to fail and disallow merging! I've gone ahead and done that. The label can be removed when ready!

**Comment by jrodewig** [comment]
> @jrodewig Adding the `pr/do-not-merge` label will cause checks to fail and disallow merging! I've gone ahead and done that. The label can be removed when ready!

Great tip — Thanks @ptpaterson!

**Review by pnwpedro — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -443,6 +437,136 @@ const client = new Client({ http2_session_idle_ms: 6000 });
...
+### Start a stream
+
+To get a stream token, append
+[`toStream()`](https://docs.fauna.com/fauna/current/reference/reference/schema_entities/set/tostream)
+or
+[`changesOn()`](https://docs.fauna.com/fauna/current/reference/reference/schema_entities/set/changeson)
+to a set from a [supported
+source](https://docs.fauna.com/fauna/current/reference/streaming_reference/#supported-sources).
+
+To start and subscribe to the stream, pass the stream token to `Client.stream()`:
```
Comment (w01fgang) [inline README.md:453]: is it a `Client` class or its instance `client`?

File: README.md [hunk 2]
```diff
@@ -443,6 +437,136 @@ const client = new Client({ http2_session_idle_ms: 6000 });
...
+    initialPage: set.pageSize(10),
+    streamToken: set.toStream()
+  }
+`);
+const { initialPage, streamToken } = response.data;
+
+client.stream(streamToken)
+```
+
+You can also pass a query that produces a stream token directly to `Client.stream()`:
```
Comment (w01fgang) [inline README.md:469]: same here, should it be `client.stream()`?

File: README.md (hunk 1 above)
Comment (jrodewig) [reply README.md:453]: It's an instance.

File: README.md (hunk 2 above)
Comment (jrodewig) [reply README.md:469]: Yes — It's an instance here as well.

**Pull request merged**